### PR TITLE
Add a reworked dedicated page for "how to create a PR"

### PR DESCRIPTION
**Pull request by matks**

### CODE REVIEW

File: src/content/1.7/contribute/contribute-pull-requests/create-pull-request.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+```
+
+> You will need to use your GitHub credentials.
+
+Now you can create your Pull Request on GitHub.
+
+If you do not know how to do it, you can read [GitHub documentation](https://help.github.com/articles/creating-a-pull-request/).
+
+If you find this process quite complex, the following materials can help you (on dev.to):
+- [The SIMPLEST way to make a pull request](https://dev.to/lukegarrigan/the-simplest-way-to-make-a-pull-request-2h61)
```
Comment (matks) [inline src/content/1.7/contribute/contribute-pull-requests/create-pull-request.md]: I added 2 helpful links

File: src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md [hunk 2]
```diff
@@ -97,38 +97,8 @@ You can execute it in your dockerized PrestaShop application without altering yo
 docker exec prestashop-git sh tests/check_phpunit.sh
 ```
 
-### Publish your contribution on GitHub
+### Publish your work
 
-Once your changes sound good and tests pass on your local computer, it's time to publish your work online and make a pull request.
+See [next page]({{</* ref "1.7/contribute/create-pull-request.md" */>}})
```
Comment (eternoendless) [inline src/content/1.7/contribute/contribute-pull-requests/contribute_using_docker.md]: What's up with the `/* ... */` ?

File: src/content/1.7/contribute/contribute-pull-requests/create-pull-request.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+---
+title: Submit a Pull Request
+weight: 4
+---
+
+# Publish your contribution on GitHub
+
+Once your changes sound good and tests pass on your local computer, you can submit these changes as a Pull Request on GitHub in order to contribute to the opensource project.
```
Comment (eternoendless) [inline src/content/1.7/contribute/contribute-pull-requests/create-pull-request.md]:
```suggestion
Once your changes sound good and tests pass on your local computer, you can contribute to the open source project by submitting these changes as a Pull Request on GitHub.
```

File: src/content/1.7/contribute/contribute-pull-requests/create-pull-request.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+First you need to push your branch on GitHub:
+
+```
+git push origin add-emoticons-support
+```
+
+> You will need to use your GitHub credentials.
```
Comment (eternoendless) [inline src/content/1.7/contribute/contribute-pull-requests/create-pull-request.md]:
```suggestion
{{% notice note %}}
You will need to use your GitHub credentials.
{{% /notice %}}
```

File: src/content/1.7/contribute/contribute-pull-requests/create-pull-request.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+Now you can create your Pull Request on GitHub.
+
+If you do not know how to do it, you can read [GitHub documentation](https://help.github.com/articles/creating-a-pull-request/).
```
Comment (eternoendless) [inline src/content/1.7/contribute/contribute-pull-requests/create-pull-request.md]:
```suggestion
Now you can create your Pull Request on GitHub. If you don't know how to do it, you can read [GitHub documentation](https://help.github.com/articles/creating-a-pull-request/).
```

File: src/content/1.7/contribute/contribute-pull-requests/create-pull-request.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+If you find this process quite complex, the following materials can help you (on dev.to):
```
Comment (eternoendless) [inline src/content/1.7/contribute/contribute-pull-requests/create-pull-request.md]:
```suggestion
If you find this process quite complex, the following articles can help you:
```

I don't think it's useful to cite the source if you're linking to it.

File: src/content/1.7/contribute/contribute-pull-requests/create-pull-request.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+- [The SIMPLEST way to make a pull request](https://dev.to/lukegarrigan/the-simplest-way-to-make-a-pull-request-2h61)
+- [The github workflow explained](https://dev.to/mathieuks/introduction-to-github-fork-workflow-why-is-it-so-complex-3ac8)
+
+> Do not forget to complete the contribution table, this is really important for the Core Team to really understand what is the value of your contribution.
```
Comment (eternoendless) [inline src/content/1.7/contribute/contribute-pull-requests/create-pull-request.md]: Maybe link to the contribution guidelines instead?

File: src/content/1.7/contribute/contribute-pull-requests/create-pull-request.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+
+
+## Syncing your fork
+
+PrestaShop Core is a really active project with more than 30 contributions accepted per week, so your copy will become outdated
+really fast. To make your own copy up to date with the original project, only a few commands are required:
```
Comment (eternoendless) [inline src/content/1.7/contribute/contribute-pull-requests/create-pull-request.md]:
```suggestion
PrestaShop Core is a really active project with more than 30 contributions accepted per week, so your fork will become outdated really fast. To make your own copy up to date with the original project, only a few commands are required:
```

File: src/content/1.7/contribute/contribute-pull-requests/create-pull-request.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+
+> You need to execute these commands at the root of your copy/fork.
```
Comment (eternoendless) [inline src/content/1.7/contribute/contribute-pull-requests/create-pull-request.md]:
```suggestion
{{% notice note %}}
You need to execute these commands at the root of your copy/fork.
{{% /notice %}}
```

**Comment by matks** [comment]
Applied @eternoendless feedbacks in https://github.com/PrestaShop/docs/pull/471/commits/bfba62f3a2ca036f6b139297940f6cfe371ef0bb

**Pull request merged**